syntax = "proto3";

package SmartHomeBuddy;

message DbLoadRequest {
  string file_path = 1;
  string file_type = 2;
}

message DbLoadResponse {
  bool is_done = 1;
}

service PcapDatabase {
  rpc LoadPcapIntoDatabase(DbLoadRequest) returns (DbLoadResponse);
}